packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -402,6 +402,11 @@ class _AnyTapGestureRecognizer extends BaseTapGestureRecognizer {
 
   @override
   String get debugDescription => 'any tap';
+
+  @override
+  void handleTapMove({required PointerMoveEvent move}) {
```

Comment:
If you provided a default implementation for handleTapMove, then you wouldn't need this. Not sure if that's good or not though.